src/thor/multimodal.cc

```diff
@@ -44,8 +44,8 @@ constexpr uint32_t kInitialEdgeLabelCount = 200000;
 MultiModalPathAlgorithm::MultiModalPathAlgorithm(const boost::property_tree::ptree& config)
     : PathAlgorithm(config.get<uint32_t>("max_reserved_labels_count", kInitialEdgeLabelCount),
                     config.get<bool>("clear_reserved_memory", false)),
-      walking_distance_(0), max_label_count_(std::numeric_limits<uint32_t>::max()),
-      mode_(travel_mode_t::kPedestrian), travel_type_(0) {
+      max_label_count_(std::numeric_limits<uint32_t>::max()), mode_(travel_mode_t::kPedestrian),
```

Comment:
`walking_distance_` was only used in `ExpandForward`, similar to `new_cost`, no need to have it as a class member, was kinda confusing at first.

Reply:
we put it now in the MMEdgeLabel